### DISCUSSION TITLE
updating the logo

### DISCUSSION THREAD

**Discussion by wild-fruits** [bevyengine/bevy#13485] [Ideas]
What about updating the logo? Has anyone thought about creating a logo design competition for version 0.14?

**Comment by rparrett** [comment]
There's another open discussion where logo ideas have been iterated on, but:

> I am still very happy with the current logo, but I'm also happy to provide an outlet for people to express themselves. Every so often someone wants to take a whack at it (there have been some semi-recent iterations on discord and twitter). And theres always a chance that someone will propose something that should clearly be "the next bevy logo".
> -- cart

https://github.com/bevyengine/bevy/discussions/9527#discussioncomment-6786148

**Reply by alice-i-cecile** [reply]
Agreed: I think the logo is honestly very good and has a solid brand identity. There *are* elements of our marketing that need love (hi home page), and I'd rather spend effort there :)
